forge-gui/res/cardsfolder/upcoming/aether_spike.txt

```diff
@@ -0,0 +1,11 @@
+Name:Aether Spike
+ManaCost:1 U
+Types:Instant
+A:SP$ Pump | TargetType$ Spell | TgtZone$ Stack | ValidTgts$ Card | SubAbility$ DBPutCounter | StackDescription$ {p:You} chooses {s:Targeted}. | SpellDescription$ Choose target spell.
+SVar:DBPutCounter:DB$ PutCounter | Defined$ You | AILogic$ PayEnergy | CounterType$ ENERGY | CounterNum$ 2 | SubAbility$ DBChooseNumber | SpellDescription$ You get {E}{E} (two energy counters), then you may pay any amount of {E}. 
+SVar:DBChooseNumber:DB$ ChooseNumber | Max$ Max | ListTitle$ Pay Energy to counter targeted spell | SubAbility$ DBPay
+SVar:DBPay:DB$ Pump | UnlessCost$ PayEnergy<X> | UnlessPayer$ You | UnlessSwitched$ True | SubAbility$ DBCounter
+SVar:DBCounter:DB$ Counter | Defined$ Targeted | UnlessCost$ X | UnlessPayer$ TargetedController | SpellDescription$ Counter that spell unless its controller pays [1} for each {E} paid this way. 
```

Comment:
that's not a X shard
use a different char

Reply:
I don't think I understand the problem ?

Reply:
Did you test your changes?
I have doubts since you introduced xPaid now...

Reply:
does it work if you only use N for everything?

Reply:
I think so ? 
![image](https://github.com/Card-Forge/forge/assets/67333662/d2c919a9-ecd6-4931-adbe-5b412cf49e29)

Reply:
now just fix that StackDescription ;)